modules/script.js

```diff
@@ -0,0 +1,94 @@
+import Books from './displayBook.js';
+import { DateTime } from './luxon/src/luxon.js';
+
+const allBooks = new Books([]);
+
+function addToLocalStorage() {
+  const stringifyArray = JSON.stringify(allBooks.bookArray);
+  localStorage.setItem('storedBooks', stringifyArray);
+}
```

Comment:
- [ ] good job refactoring :clap: , Please notice that one of the main goals of this project is to refactor it to use ES6 syntax, which uses arrow functions instead, I suggest we change all the functions to be arrow functions :pray: